doc/collections/_adrs/0002-decision-to-use-monorepos.md

```diff
@@ -1,4 +1,4 @@
-## ADR 001: Decision to Use Monorepos
+## ADR 0002: Decision to Use Monorepos
```

Comment:
```suggestion
## ADR 0002: Numbering and Organization
```

would be nice if we could ♻️ our num. 2 for content of AIP 0002 mdash; Numbering and Organization mdash; https://aep.dev/2

Reply:
determining how we have organized and created a sensible knowledge-base-like system
should probably _not_ be an ADR here &mdash; it would look like lack of foresight !